OxfordBodleian/Juel-Jensen/BDLaethd19.xml

```diff
@@ -202,35 +217,65 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+
                             </handNote>
                         </handDesc>
-                        
+
                        <!-- <decoDesc>
-                         
+
                         </decoDesc>-->
-                        
-                     <!--   <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
+
+                        <bindingDesc>
+                            <binding contemporary="true Ethiopian">
+                              <decoNote xml:id="b1" type="bindingMaterial">
+                                The book is bound in <material key="wood">wooden</material> boards with a blind-tooled <material
+                                key="leather">leather</material> which covers only the spine and the back edge of the boards.
+                              </decoNote>
+                              <decoNote xml:id="b2" type="Boards">
+                                The boards are flush with the bookblock and have external shallow bevels. The grain runs vertically.
+                                The outer surface and the edges of the boards are polished and a layer of dark-coloured finish has been added, while the inner surface exhibits signs of the working tool.
+                                The lower board is separated vertically in two halves which are held together with various threads passing through pairs of holes drilled in the board.
+                                On the inner surface of the upper board, in the middle, remnants of a grayish substance are present, which perhaps was used to glue a <term key="mirror">mirror</term>.
+                              </decoNote>
+                              <decoNote xml:id="b3" type="Cover" color="reddish-brown">
+                                The boards and the spine are partially covered with a reddish-brown leather featuring a blind-tooled decoration.
+                                On the surface of the boards and on the back, bands of two <term key="singleStraightLine">single straight lines</term> intersect
+                                forming two columns of juxtaposed panels. Within each panel, the lines cross each other forming a crux decussata. Within a panel
+                                on the lower boards the lines also form a 4-pointed star.
+                                The turn-ins and the edges are not decorated. On the inner side of the boards, between the turn-ins, <term key="additionalLeatherPatch">additional leather patches</term>
+                                have been added. On the one on the upper board, an angel <!-- is it an angel? -->is drawn in pencil.
```

Comment:
Perhaps St Michael? I see he holds a spear

Reply:
I don't think you can identify angels in ethiopian art just by their attributes :(